app/src/main/java/com/keylesspalace/tusky/util/Either.kt

```diff
@@ -45,3 +45,6 @@ sealed class Either<out L, out R> {
         }
     }
 }
+
+fun <V> V.left(): Either<V, Nothing> = Either.Left(this)
```

Comment:
maybe `asLeft()` ? otherwise it looks like getter

Reply:
Ah, I did not end up using those extensions.